vscode/src/client/analyzerClient.ts

```diff
@@ -649,31 +647,10 @@ export class AnalyzerClient {
    * rulesets yaml files to provide to the analyzer.  After the issue is resolve, send all
    * of the rulesets directories either as `string[]` or as a joined list.
    */
-  public getRulesetsPath(): string {
-    const includedRulesets = this.assetPaths.rulesets;
-
-    // TODO(djzager): konveyor/kai#509
-    // const useDefaultRulesets = getConfigUseDefaultRulesets();
-    // const customRules = getConfigCustomRules();
-    // const rules: string[] = [];
-
-    // if (useDefaultRulesets) {
-    //   rules.push(includedRulesets);
-    // }
-    // if (customRules.length > 0) {
-    //   rules.push(...customRules);
-    // }
-    // return rules;
-
-    return includedRulesets;
-  }
-
-  // New method to retrieve stored rulesets
-  public getStoredRulesets(): RuleSet[] | null {
-    if (this.extContext) {
-      const storedRulesets = this.extContext.globalState.get("storedRulesets");
-      return storedRulesets ? JSON.parse(storedRulesets as string) : null;
-    }
-    return null;
+  public getRulesetsPath(): string[] {
+    return [
+      getConfigUseDefaultRulesets() && this.assetPaths.rulesets,
+      ...getConfigCustomRules(),
+    ].filter((value): value is string => typeof value === "string");
```

Comment:
Pfffttt!! I forgot about the hack in tackle2-ui that fixes that auto-typing BS.

I'll fix that up in a followup because it'll annoy me forever until I fix it.

For now, an alternate form would be:
```ts
  ].filter(Boolean) as string[];
```